helios-services/src/main/java/com/spotify/helios/servicescommon/KafkaSender.java

```diff
@@ -39,7 +39,12 @@ public KafkaSender(final Optional<KafkaProducer<String, byte[]>> kafkaProducer)
     this.kafkaProducer = kafkaProducer;
   }
 
-  public void send(final KafkaRecord kafkaRecord) {
+  @Override
+  public boolean isHealthy() {
+    return true;
```

Comment:
Unrelated to PR, but should we always return true here? Is there value in checking we can talk to kafka by using `KafkaProducer.waitOnMetadata() for example?